container_pipeline/lib/index.py

```diff
@@ -0,0 +1,82 @@
+import os
+from glob import glob
+from container_pipeline.utils import load_yaml
+
+import container_pipeline.constants.index
+import logging
+import yaml
+
+
+class IndexEntry(object):
```

Comment:
index reader wont help here? so that we have a common logic everywhere. in case of updating something it gets updated everywhere?

Reply:
Common login should be in lib or utils. That is why i put it there. Also the indexentry thing means you wont be dealing with dict keys and hence less runtime error prone.